web/templates/premium.html

```diff
@@ -40,126 +40,94 @@
           </p>
         </center>
-        <div id="months-slider" style="margin: 0 auto; max-width: 500px"></div>
-        <br />
-        <div id="cost" class="centered"><b>{{ $.T "Loading..." }}</b></div>
-        <br />
         <div
           class="ui internally celled two column center aligned stackable grid">
           <div class="column">
             <h2 class="ui header">
               <i class="fa-brands fa-paypal"></i>
               {{ $.T "PayPal" }}
             </h2>
-            <form
-              id="paypal-form"
-              action="https://www.paypal.com/cgi-bin/webscr"
-              method="post"
-              target="_self">
-              <input type="hidden" name="on0" value="Period" />
-              <input type="hidden" name="os0" value="1 month" />
-              {{ range _range 24 }}
-                {{ $months := plus (float .) 1 }}
-                <input
-                  type="hidden"
-                  name="option_select{{ . }}"
-                  value="{{ $months }} month{{ if . }}s{{ end }}" />
-                <input
-                  type="hidden"
-                  name="custom"
-                  value="userid={{ $global.User.ID }}" />
-                <input
-                  type="hidden"
-                  name="option_amount{{ . }}"
-                  value="{{ calculatePremiumPrice $months }}" />
-              {{ end }}
-              <input
-                type="hidden"
-                name="on1"
-                value="Akatsuki user to give premium:" />
-              <input
-                type="hidden"
-                name="amount"
-                value="{{ calculatePremiumPrice 1 }}"
-                id="paypal-amt" />
-              <div class="ui fluid labeled action input">
-                <div class="ui label">
-                  {{ $.T "User:" }}
-                </div>
-                <input
-                  id="username-input"
-                  name="os1"
-                  type="text"
-                  placeholder="{{ $.T "User" }}"
-                  value="{{ $global.User.Username }}" />
-                <button type="submit" class="ui blue button">
-                  {{ $.T "Submit" }}
-                </button>
-              </div>
-              {{ $ief }}
-              <input
-                type="hidden"
-                name="business"
-                value="{{ config "PAYPAL_EMAIL_ADDRESS" }}" />
-              <input type="hidden" name="cmd" value="_xclick" />
-              <input type="hidden" name="lc" value="GB" />
-              <input type="hidden" name="no_note" value="0" />
-              <input type="hidden" name="currency_code" value="EUR" />
-              <img
-                alt=""
-                border="0"
-                src="https://www.paypalobjects.com/it_IT/i/scr/pixel.gif"
-                width="1"
-                height="1" />
+
+            <form action="https://www.paypal.com/cgi-bin/webscr" method="post">
```

Comment:
<img width="302" alt="image" src="https://github.com/user-attachments/assets/c5dd498d-f31a-4282-8e97-4c5d19b3ec5f">

Reply:
I still have some work to do for:
1. Being able to purchase premium for others
2. Being able to purchase multiple months at a time
3. Have proper logic to cancel subscription when it runs out and we don't have card on file